src/Shared/Nullable/NullableAttributes.cs

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// From https://github.com/dotnet/runtime/blob/ef72b95937703e485fdbbb75f3251fedfd1a0ef9/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>Specifies that null is allowed as an input even if the corresponding type disallows it.</summary>
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property, Inherited = false)]
+#if INTERNAL_NULLABLE_ATTRIBUTES
```

Comment:
Am I correct these attributes will be `public` when targeting .NET 5 and `internal` otherwise?

Reply:
This file doesn't get used in .NET Core since these types are already part of the runtime and we get them for free. They should be available as internal types in .NET Core. I copied this file as-is from dotnet/runtime with the hope that it would be easier to keep it updated as they add new annotations.